evol/helpers/_utils.py

```diff
@@ -10,6 +10,6 @@ def select_partition(length: int, min_size: int=1, max_size: int=None) -> Tuple[
     :param max_size: Maximum length of the partition. Defaults to length - 1.
     :return: Start and end index of the partition.
     """
-    partition_size = randint(min_size, length-1 if max_size is None else max_size)
+    partition_size = randint(min_size, max_size or length - 1)
```

Comment:
Although this is less characters, I find it less obvious that this does what it is supposed to do when reading it. What is the main reason for making this change?

Reply:
I do like it; it's much more compact than the original, although less explicit.